doc/source/user_guide/io.rst

```diff
@@ -757,6 +757,7 @@ the end of each data line, confusing the parser. To explicitly disable the
 index column inference and discard the last column, pass ``index_col=False``:
 
 .. ipython:: python
+    :okwarning:
```

Comment:
If this is going to warn, should the docs here then have to be updated to reflect this change?

(but is this actually going to warn? Below I read "One set of trailing commas is allowed.", which is the case here?)

Reply:
Good point, this raised a Warning earlier before allowing one set of trailing commas